keyboards/foxlab/leaf80/rules.mk

```diff
@@ -0,0 +1,23 @@
+# MCU name
+MCU = atmega32a
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = bootloadHID
+
+# build options
+BOOTMAGIC_ENABLE = no
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = yes
+CONSOLE_ENABLE = no
+COMMAND_ENABLE = no
+
+BACKLIGHT_ENABLE = no
```

Comment:
From the PCB photo it is obvious that it has backlight, just figuring out `BACKLIGHT_PIN` is needed (most likely it is `D4`, like on other bootmapper PCBs). Then this feature should be enabled, so that it would be available for users of config.qmk.fm.